_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -0,0 +1,300 @@
+
+- **unloaded img**: the element is an `img` and its [current request][]'s [state][image request state] is not [completely available][]; or
+- **ignored svg**: the element is an `svg` with an empty (`""`) [accessible name][] and a [semantic role][] of `graphics-document`; or
+- **ignored canvas**: the element is a `canvas` with an empty (`""`) [accessible name][] and no [explicit semantic role][]; or
+- **named from author**: the element is a [descendant][] in the [flat tree][] of an element that is [named from author][].
+
+**Note**: An example of an image ignored because of "named from author" is when the image is a descendant of a `button` element that uses `aria-label` for its accessible name.
+
+## Expectation
+
+Each test target is [purely decorative][].
+
+**Note**: It is relatively common for an informative image such as an icon to be marked up as decorative, if the text alternative is adjacent to the image. This is a [conforming alternative version][] for the image. This fails the rule but meets [conformance requirement 1 of WCAG 2.1](https://www.w3.org/TR/WCAG21/#cc1).
+
+## Assumptions
+
+- `svg` elements with a [semantic role][] of `graphics-document` and with an empty (`""`) [accessible name][] are ignored by assistive technologies tested for this rule. If some assistive technology does not ignore these elements, and that assistive technology is required for conformance, passing this rule does not ensure all decorative `svg` elements can be ignored, and the [success criterion 1.1.1 Non-text content][] may still not be satisfied. The same is true for `canvas` elements with no [semantic role][] and an empty (`""`) [accessible name][].
+
+- A web page with informative images without an [accessible name][] may conform to WCAG 2.1 Level A when the information provided by that image is available elsewhere, either on the web page itself or on a [conforming alternative version][] of the page. For example if an equivalent text is adjacent to the image, or if the text alternative is included in the [accessible name][] of a parent element.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [H67: Using null alt text and no title attribute on img elements for images that AT should ignore](https://www.w3.org/WAI/WCAG21/Techniques/html/H67.html)
+- [Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `img` element with an empty `alt` attribute which is not included in accessibility tree is decorative.
+
+```html
+<p>Happy new year!</p>
+<img src="/test-assets/shared/fireworks.jpg" alt="" />
+```
+
+#### Passed Example 2
+
+This `img` element that is ignored by assistive technologies because `aria-hidden` is set to `true` is [purely decorative][].
```

Comment:
I'd like to have example descriptions stick closer to the applicability/expectation.
```suggestion
This `img` element, that is [visible][] and not [included in the accessibility tree][] (because `aria-hidden` is set to `true`), is [purely decorative][].
```
This goes for more or less all passed examples descriptions.

Reply:
I don't think this is necessary. We don't need to repeat the language from the applicability over and over in the example descriptions. The short version is it gets ignored by AT, which is what it says.